realm/realm-annotations-processor/src/test/resources/io/realm/some_test_AllTypesRealmProxy.java

```diff
@@ -2754,4 +2977,4 @@ public boolean equals(Object o) {
 
         return true;
     }
-}
+}
```

Comment:
```suggestion
}

```

Minor: We always end all files with a newline